src/lib/cloud-manager-hoc.jsx

```diff
@@ -96,6 +98,11 @@ const cloudManagerHOC = function (WrappedComponent) {
                 this.connectToCloud();
             }
         }
+        handleExtensionAdded (categoryInfo) {
```

Comment:
I think this approach should work great. Or... now that you've moved the `props.vm.extensionManager.isExtensionLoaded` check into `props.canModifyCloudData`, maybe we can get away with just waiting for `componentDidUpdate()` to be called? I wonder, what could we do so that adding an extension would trigger this HOC to update?

Reply:
*Updating with a separate converation*

I investigated some options with triggering a re-render. There were some complications. One interesting issue is that `this.props.vm.extensionManager.isExtensionLoaded('videoSensing')` is still false at the point of the `EXTENSION_ADDED` callback being called, making it challenging to use `canModifyCloudData`. I will add a comment to the code to this effect.